compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/window/ApplicationStateListener.uikit.kt

```diff
@@ -26,14 +26,16 @@ import platform.UIKit.UIApplicationWillEnterForegroundNotification
 import platform.darwin.NSObject
 
 internal class ApplicationStateListener(
+    /**
+     * [NSNotificationCenter] to listen to, can be customized for tests purposes
+     */
+    private val notificationCenter: NSNotificationCenter = NSNotificationCenter.defaultCenter,
```

Comment:
```suggestion
    private val notificationCenter get() = NSNotificationCenter.defaultCenter,
```
No need to store it

Reply:
It's a constructor parameter, which is mocked in tests

Reply:
This

Reply:
Ah, right